src/Environment/PathProvider.cs

```diff
@@ -60,16 +62,44 @@ public static string RootDirectory
             if (devRootPath is not null)
                 return devRootPath;
 
+            var rootPath = "/";
+
             switch (OsInfo.CurrentOS)
```

Comment:
Would it maybe be a good idea to pass in the operating system as a ENV variable instead of trying to determine it ourselves? The current determination is really hacky and was stolen from how Sonarr does it, but a better way should definitely be possible.

Reply:
Isn't there some library that realibly can? I think we're going to need more is info if we want to go towards prebuilds and if we use a prebuild, we already know which os is supported because we have build for that. Should already be available maybe?

Reply:
I have found something that could work, created a separate issue for this: https://github.com/PlexRipper/PlexRipper/issues/273